keyboards/keyten/kt3700/kt3700.h

```diff
@@ -0,0 +1,54 @@
+ /* Copyright 2022 Ivan Gromov (@key10iq)
+  * 
+  * This program is free software: you can redistribute it and/or modify 
+  * it under the terms of the GNU General Public License as published by 
+  * the Free Software Foundation, either version 2 of the License, or 
+  * (at your option) any later version. 
+  * 
+  * This program is distributed in the hope that it will be useful, 
+  * but WITHOUT ANY WARRANTY; without even the implied warranty of 
+  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+  * GNU General Public License for more details. 
+  * 
+  * You should have received a copy of the GNU General Public License 
+  * along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+  */ 
+
+#pragma once
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+/* LAYOUT_numpad_6x4
+ * ┌───┬───┬───┬───┐
+ * │00 │01 │02 │03 │
+ * ├───┴───┴───┴───┤
+ * │               │
+ * ├───┬───┬───┬───┤
+ * │10 │11 │12 │13 │   
+ * ├───┼───┼───┼───┤
+ * │20 │21 │22 │23 │
+ * ├───┼───┼───┼───┤
+ * │30 │31 │32 │33 │
+ * ├───┼───┼───┼───┤
+ * │40 │41 │42 │43 │
+ * ├───┼───┼───┼───┤
+ * │50 │51 │52 │53 │
+ * └───┴───┴───┴───┘
+ */
+#define LAYOUT_numpad_6x4( \
```

Comment:
This isnt `numpad_6x4`, all references should be updated to `LAYOUT_ortho_6x4` or just `LAYOUT`

Reply:
The PCB photo shows that the common `numpad_6x4` layout is supported too, therefore both `LAYOUT_numpad_6x4` and `LAYOUT_all` layout macros should be defined (and also the `LAYOUT_ortho_6x4` alias for `LAYOUT_all`).

Alternatively, the layout matrix data could be added to `info.json` instead of defining the macros in C.